src/lootthread.cpp

```diff
@@ -59,17 +59,14 @@ void LOOTWorker::setGame(const std::string& gameName)
       {"fallout3", loot::GameId::fo3},     {"fallout4", loot::GameId::fo4},
       {"fallout4vr", loot::GameId::fo4vr}, {"falloutnv", loot::GameId::fonv},
       {"skyrim", loot::GameId::tes5},      {"skyrimse", loot::GameId::tes5se},
-      {"skyrimvr", loot::GameId::tes5vr},
-  };
+      {"skyrimvr", loot::GameId::tes5vr},  {"nehrim", loot::GameId::nehrim},
+      {"enderal", loot::GameId::enderal},  {"enderalse", loot::GameId::enderalse}};
```

Comment:
At some point it might be nice to have these be passed by MO2 in some way, and these would be declared from the games themselves, but it's a lot of trouble just to remove a few game specific lines of code from the repo.